pkg/cmd/eventlistener/testdata/TestListEventListener-Multiple_EventListener.golden

```diff
@@ -1,5 +1,6 @@
-NAME   AGE
-tb1    2 minutes ago
-tb2    30 seconds ago
-tb3    1 week ago
-tb4    ---
+NAME   AGE              AVAILABLE
```

Comment:
Not that it should happen in this pr, but it might be worth looking into in future if it would be helpful to support displaying the reason like `kubectl` does:

```
NAME                          ADDRESS                                                                           AVAILABLE   REASON
github-listener-interceptor   http://el-github-listener-interceptor.petclinic-pipeline.svc.cluster.local:8080   True        MinimumReplicasAvailable
```

Reply:
Do you want me to add this reason in this PR or should raise another one?

Reply:
Let's wait and maybe open an issue to discuss it further.